tests/InputFiles/Elasticity/BentBeam.yaml

```diff
@@ -18,11 +18,13 @@ Background:
     Length: 2.
     Height: 1.
     BendingMoment: 1.
-    Material:
-      # Iron
+    Material: &iron
```

Comment:
Q) Why do we have `&` and `*` for `iron` keyword in this input file?

Reply:
This is very useful YAML syntax. See https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/ for an explanation